packages/prisma/seed.ts

```diff
@@ -1096,51 +1096,34 @@ async function main() {
             },
           ],
         },
-        {
-          memberData: {
-            email: "member1-acme@example.com",
-            password: {
-              create: {
-                hash: "member1-acme",
-              },
-            },
-            username: "member1-acme",
-            name: "Member 1",
-          },
-          orgMembership: {
-            role: "MEMBER",
-            accepted: true,
-          },
-          orgProfile: {
-            username: "member1",
-          },
-          inTeams: [
-            {
-              slug: "team1",
-              role: "ADMIN",
-            },
-          ],
-        },
-        {
+        ...Array.from({ length: 10 }, (_, i) => ({
```

Comment:
Creates 10 members . Feel free to change length to anything you want to test orgs